### ISSUE TITLE
Unhandled Exception when utility solution file is run from within Visual Studio 2015

### ISSUE THREAD

**Issue by jtipeetoes** [JimDaly/CRMJSWebAPIServiceUtil#22]
This post should be considered a virtual duplicate of issue #21 with the only difference being that the method used to run the utility in this case is opening the solution file within Visual Studio 2015 and the level of detail surrounding the exception itself. 

**Steps to recreate** 
1. Download the utility from github
2. Unblock the zip file contents using the properties menu
3. Open the resulting solution file path = CRMJSWebAPIServiceUtil-master\CRMJSWebAPIServiceUtil-master\CRMJSWebAPIServiceUtil.sln
4. Hit the 'Login to CRM' button on the utility screen
5. Hit F5 from inside of Visual Studio 2015 to run the solution 
6. Enter connection information into the resulting dialog window
![image](https://cloud.githubusercontent.com/assets/1518694/22349155/7ddd937c-e3dc-11e6-891c-98b38b929c07.png)

**Results in** 

System.Exception was unhandled by user code
  HResult=-2146233088
  Message=Error retrieving solutions: NotImplemented
  Source=CRMJSWebAPIServiceUtil
  StackTrace:
       at CRMJSWebAPIServiceUtil.MainWindow.<retrieveSolutions>d__31.MoveNext() in G:\CRM\CRMJSWebAPIServiceUtil-master\CRMJSWebAPIServiceUtil-master\CRMJSWebAPIServiceUtil\CRMJSWebAPIServiceUtil\MainWindow.xaml.cs:line 282
    --- End of stack trace from previous location where exception was thrown ---
       at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
       at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
       at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
       at CRMJSWebAPIServiceUtil.MainWindow.<LoginButton_Click>d__30.MoveNext() in G:\CRM\CRMJSWebAPIServiceUtil-master\CRMJSWebAPIServiceUtil-master\CRMJSWebAPIServiceUtil\CRMJSWebAPIServiceUtil\MainWindow.xaml.cs:line 197
  InnerException: 

**NuGet Package Information**
AjaxMin - installed version 5.14.5506.26202
Newtonsoft.Json - installed version 9.0.1
Microsoft.IdentityModel.Clients.ActiveDirectory - 2.28.1

_Packages.config screenshot_
![image](https://cloud.githubusercontent.com/assets/1518694/22349125/5931e28a-e3dc-11e6-8e6f-e1e1d741c623.png)

**Comment by JimDaly** [comment]
Applied pull request #23 which I believe will fix this issue.
Please sync and test again. Let me know if it worked and we can close this issue.
